packages/fx-core/src/common/local/taskDefinition.ts

```diff
@@ -0,0 +1,242 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+"use strict";
+
+import { FolderName, ProgrammingLanguage, npmInstallCommand } from "./constants";
+import path from "path";
+import { isWindows } from "../deps-checker/util/system";
+
+export interface ITaskDefinition {
+  name: string;
+  command: string;
+  cwd: string | undefined;
+  isBackground: boolean;
+  isShell: boolean;
+  isCmd?: boolean;
+  args?: string[];
+  env?: { [key: string]: string };
+}
+
+export class CustomTaskDefinition {
+  static command(isWatchTask: boolean): string {
```

Comment:
Seems `command` and `nameSuffix` are not used out of this file, may consider to declare them as private.

Reply:
Thx